xbmc/cores/RetroPlayer/RetroPlayerVideo.cpp

```diff
@@ -30,18 +30,22 @@ CRetroPlayerVideo::CRetroPlayerVideo(CRPRenderManager& renderManager, CRPProcess
   m_renderManager(renderManager),
   m_processInfo(processInfo)
 {
+  CLog::Log(LOGDEBUG, "RetroPlayer[VIDEO]: Initializing video");
+
   m_renderManager.Initialize();
 }
 
 CRetroPlayerVideo::~CRetroPlayerVideo()
 {
+  CLog::Log(LOGDEBUG, "RetroPlayer[VIDEO]: Deinitializing video");
+
   CloseStream();
   m_renderManager.Deinitialize();
 }
 
 bool CRetroPlayerVideo::OpenPixelStream(AVPixelFormat pixfmt, unsigned int width, unsigned int height, unsigned int orientationDeg)
 {
-  CLog::Log(LOGINFO, "RetroPlayerVideo: Creating video stream with pixel format: %i, %dx%d", pixfmt, width, height);
+  CLog::Log(LOGDEBUG, "RetroPlayer[VIDEO]: Creating video stream %ux%u", width, height);
```

Comment:
you could keep the pixel format here using `TranslatePixelFormat(AVPixelFormat format)`